DataRepo/management/commands/load_compounds.py

```diff
@@ -1,27 +1,57 @@
-from csv import DictReader
-
+import pandas as pd
 from django.core.management import BaseCommand
 
-from DataRepo.models import Compound
+from DataRepo.utils import CompoundsLoader
 
 
 class Command(BaseCommand):
     # Show this when the user types help
     help = "Loads data from a compound list into the database"
 
     def add_arguments(self, parser):
-        parser.add_argument("compound_list_filename", type=str)
+        parser.add_argument(
+            "--compounds",
+            type=str,
+            help="Path to tab-delimited file containing headers of 'Compound','Formula''HMDB ID', and 'Synonyms'; "
+            "required.",
+            required=True,
+        )
+
+        parser.add_argument(
+            "--synonym-separator",
+            type=str,
+            help="Character separating multiple synonyms in 'Synonyms' column;  defaults to semi-colon",
```

Comment:
You can use the built in `$(default)s` parameter to display the default value in help messages.
```suggestion
            help="Character separating multiple synonyms in 'Synonyms' column (default '%(default)s'",
```
will display
```
  --synonym-separator SYNONYM_SEPARATOR
                        Character separating multiple synonyms in 'Synonyms'
                        column (default ';').
```